packages/pangea-sdk/src/main/java/cloud/pangeacyber/pangea/audit/AuditClient.java

```diff
@@ -180,6 +203,7 @@ public LogResponse log(Event event) throws PangeaException, PangeaAPIException{
      * @param event event to log
      * @param signMode "Unsigned" or "Local"
      * @param verbose true to more verbose response
+     * @param pkInfo additional information about public key. Will be ignored if signMode is UNSIGNED
```

Comment:
I don't see a reason why a user can supply different pkInfo for different log calls. I think it should be set in the Audit instance and use the same every time...